packages/terra-legacy-theme/src/scss/root-theme.scss

```diff
@@ -1,2 +1,3 @@
 @import 'application-menu';
+@import 'brand-footer';
```

Comment:
Should we have snapshots with this?

Reply:
Added tests in 0179d258ece4c3ca5459b19d8d5c7b932cf59be8